.circleci/config.yml

```diff
@@ -4,10 +4,20 @@ version: 2.1
 workflows:
   main:
     jobs:
-      - php72-tests
-      - php73-tests
-      - acceptance-tests
-      - a11y-tests
+      # Get new instance if none was used for the PR, else figure out which instance.
```

Comment:
I replaced the other jobs with stubs that finish immediately, to not bogart concurrent jobs.